package.json

```diff
@@ -25,10 +26,12 @@
   },
   "homepage": "https://github.com/fraserxu/electron-pdf",
   "devDependencies": {
-    "standard": "^6.0.6"
+    "jasmine": "^2.5.2",
+    "standard": "^8.4.0"
```

Comment:
I had the newer version installed globally and noticed it has a nice --fix option, so I bumped this up.